### PR TITLE
Oceanwater 704 moveit commander throws an exception for arm ros actions interface under noetic

### PR DESCRIPTION
**Pull request by Samahu**
## Linked Issues:
| EPIC ⚡| [OCEANWATER-577 / Support for ROS Noetic](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-577) |
| :----------- | :----------- |
| Jira Ticket 🎟️   | [OCEANWATER-704 / MoveIt Commander throws an exception for arm ros actions interface under Noetic](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-704) |
| Github :octocat:  | N/A |

## Summary of Changes
* The goal for all MoveIt MoveGroup planning requests is explicitly set in a separate call
* All plan requests expects a tuple of 4 as an answer to the return request    

* Run through tests [2](https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/OW-TEST008+-+release-8-0#OWTEST008release80-2.Samplecollection:planningandexecution) and [9](https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/OW-TEST008+-+release-8-0#OWTEST008release80-9.ROSActions) of [OceanWATERS release 8 tests](https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/OW-TEST008+-+release-8-0)

### CODE REVIEW

**Comment by mogumbo** [comment]
I can now unstow the arm with a service or an action, but the arm moves about 10X too slow. I fixed the same symptom here https://github.com/nasa/ow_simulator/commit/3034e5c3f0f1ef455998738dad6fc56d88f60d3f but it is happening again. Is anyone else having the same experience?

Release 8 Test 9 says to call `python unstow_action_client.py` but this results in:
`Traceback (most recent call last):
  File "unstow_action_client.py", line 9, in <module>
    import rospy
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/__init__.py", line 47, in <module>
    from std_msgs.msg import Header
  File "/opt/ros/noetic/lib/python3/dist-packages/std_msgs/msg/__init__.py", line 1, in <module>
    from ._Bool import *
  File "/opt/ros/noetic/lib/python3/dist-packages/std_msgs/msg/_Bool.py", line 6, in <module>
    import genpy
  File "/opt/ros/noetic/lib/python3/dist-packages/genpy/__init__.py", line 34, in <module>
    from . message import Message, SerializationError, DeserializationError, MessageException, struct_I
  File "/opt/ros/noetic/lib/python3/dist-packages/genpy/message.py", line 48, in <module>
    import yaml
ImportError: No module named yaml`

Instead I need to type `./unstow_action_client.py` Does anyone else have this problem?

File: ow_lander/scripts/unstow_action_server.py [hunk 1]
```diff
@@ -58,7 +58,8 @@ def _update_motion(self):
         print("Unstow arm activity started")
         goal = self._interface.move_arm.get_current_pose().pose
```
Comment (mogumbo) [inline ow_lander/scripts/unstow_action_server.py:59]: Why is goal written to here if it immediately written to on the next line?

**Review by mogumbo — CHANGES_REQUESTED** [review]

File: ow_lander/scripts/stow_action_server.py [hunk 2]
```diff
@@ -55,10 +55,11 @@ def _update_feedback(self):
         
     def _update_motion(self):
 
-        print("Unstow arm activity started")
+        print("Stow arm activity started")
         goal = self._interface.move_arm.get_current_pose().pose
         goal = self._interface.move_arm.get_named_target_values("arm_stowed")
```
Comment (mogumbo) [inline ow_lander/scripts/stow_action_server.py:60]: Same question here. Why write to `goal` twice?

**Review by mogumbo — CHANGES_REQUESTED** [review]

**Comment by Samahu** [comment]
> I can now unstow the arm with a service or an action, but the arm moves about 10X too slow. I fixed the same symptom here [3034e5c](https://github.com/nasa/ow_simulator/commit/3034e5c3f0f1ef455998738dad6fc56d88f60d3f) but it is happening again. Is anyone else having the same experience?
> 
> Release 8 Test 9 says to call `python unstow_action_client.py` but this results in:
> `Traceback (most recent call last): File "unstow_action_client.py", line 9, in <module> import rospy File "/opt/ros/noetic/lib/python3/dist-packages/rospy/__init__.py", line 47, in <module> from std_msgs.msg import Header File "/opt/ros/noetic/lib/python3/dist-packages/std_msgs/msg/__init__.py", line 1, in <module> from ._Bool import * File "/opt/ros/noetic/lib/python3/dist-packages/std_msgs/msg/_Bool.py", line 6, in <module> import genpy File "/opt/ros/noetic/lib/python3/dist-packages/genpy/__init__.py", line 34, in <module> from . message import Message, SerializationError, DeserializationError, MessageException, struct_I File "/opt/ros/noetic/lib/python3/dist-packages/genpy/message.py", line 48, in <module> import yaml ImportError: No module named yaml`
> 
> Instead I need to type `./unstow_action_client.py` Does anyone else have this problem?

I created a ticket for the slowness issue: https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-705

File: ow_lander/scripts/unstow_action_server.py (hunk 1 above)
Comment (Samahu) [reply ow_lander/scripts/unstow_action_server.py:59]: same response.. I am pretty sure the first call is not needed. I will remove it

**Review by mogumbo — APPROVED** [review]

**Comment by Samahu** [comment]
@mogumbo you are the only other person with a ROS Noetic for now so I don't expect others to be able to test this. Additionally, we can have a full test on the the main ticket OCEANWATER-610

**Pull request merged**